### ISSUE TITLE
Sort the news feed by publishing time

### ISSUE THREAD

**Issue by grep0** [MoeraOrg/moera-issues#298]
When you add a new contact, their posts are added on top of the news feed - this is not convenient as earlier posts of other contacts are buried. It would be better to have default sort by publishing time first to see the newer posts earlier.

**Comment by smelamud** [comment]
There is a problem of trust with this approach. Some users may modify the date of publications of their posts to move their posts up in the feeds of their subscribers.

Another problem will uprise when view permissions are implemented. If some user publishes a hidden post and then after several days decides to make it public, this post will be placed in the middle of the newsfeed, below all recent posts of others. The same will happen in multi-server environment, if due to connectivity problems, downtime or capacity some posts will not be downloaded immediately. They also will be placed below more recent posts and the newsfeed reader may not notice them.

**Comment by grep0** [comment]
I dislike the situation when adding a new contact spams the news feed with their old posts. There should be a horizon (~5 posts not older than ~5 days)

**Comment by smelamud** [comment]
I agree. This is a temporary measure to fill the newsfeed somehow when there are not so many posts. I plan to disable it in the future.
